src/r/recordFactories/ProjectFactory.ts

```diff
@@ -755,14 +850,24 @@ export class ProjectFactory extends RecordFactory<RT.project> {
     for(const scene of scenesFromClipboard) {
       // ! Only scenes are inserted in place when position is passed. Have to live with this assumption for now
       // * if position is passed, then keep incrementing to insert in order, else add at the end of the list
-      this.addRecord(scene, position? position++: position);
+      const addedScene = this.addRecord(scene, position? position++: position);
+      const sceneF = r.scene(addedScene);
+      const elements = sceneF.getRecords(RT.element);
```

Comment:
You don't need all the elements. 
Consider there are 200 elements in a scene (this is a realistic number from existing projects) and then think of the performance hit of the addLinkedVariables function.